docs/src/pages/ui/primitives/selectfield/react.mdx

```diff
@@ -281,6 +281,70 @@ import '@aws-amplify/ui-react/styles.css';
   </Flex>
 </Example>
 
+### Descriptive text
+
+To provide additional descriptive text of requirements of the field, use the `descriptiveText` field. It will only be visible when `label` is not hidden.
+
+```jsx
+import { SelectField } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+<SelectField
+  label="Fruit"
+  descriptiveText="A fruit of your choice"
+  placeholder="Please select a fruit"
+>
+  <option value="apple">Apple</option>
+  <option value="banana">Banana</option>
+  <option value="orange">Orange</option>
+</SelectField>;
+```
+
+<Example>
+  <SelectField
+    label="Fruit"
+    descriptiveText="A fruit of your choice"
+    placeholder="Please select a fruit"
+  >
+    <option value="apple">Apple</option>
+    <option value="banana">Banana</option>
+    <option value="orange">Orange</option>
+  </SelectField>
+</Example>
+
+### State
+
+`isDisabled` is the only available state for SelectField. A disabled field will be not be focusable not mutable and will not be submitted with form data.
```

Comment:
nit: could mention that if customers want a readonly field that's still submitted with the form, they can make all the options besides the selected option be disabled:

```
<option disabled></option>
<option value="default"></option>
<option disabled></option>
```
ref: https://stackoverflow.com/a/23428851

Reply:
I saw that as well when trying to find out if `select` has a `readonly` state!! Then I think it makes sense that `select` doesn't has that support because the use case is wired. A read-only select could totally be an `input` 🤣